doc/ale-llvm.txt

```diff
@@ -0,0 +1,19 @@
+===============================================================================
+ALE LLVM Integration                                           *ale-llvm-options*
```

Comment:
I can't tell if the tags are aligned to the right margin properly or not on GitHub, but they might not be. Make sure they are.

Reply:
I did not notice that `===` line is longer than title. I'll align titles.